packages/eslint-plugin/tests/rules/component-max-inline-declarations.test.ts

```diff
@@ -14,103 +14,68 @@ import rule, {
 });
+const messageId: MessageIds = 'componentMaxInlineDeclarations';
 
 ruleTester.run(RULE_NAME, rule, {
   valid: [
-    // should succeed if the number of lines does not exceeds the default lines limit (template)
+    // should succeed if the number of the template lines does not exceeds the default lines limit
     `
     @Component({
       template: '<div>just one line template</div>'
     })
     class Test {}
     `,
-    {
-      // should succeed if a negative limit is used and the number of lines does not exceeds the default lines limit (template)
-      code: `
-      @Component({
-        template: '<div>first line</div>'
-      })
-      class Test {}
-      `,
-      options: [
-        {
-          template: -5,
-        },
-      ],
-    },
-    // should succeed if the number of lines does not exceeds the default lines limit (styles)
+    // should succeed if the number of the styles lines does not exceeds the default lines limit
     `
     @Component({
       styles: ['div { display: none; }']
     })
     class Test {}
     `,
-    {
-      // should succeed if a negative limit is used and the number of lines does not exceeds the default lines limit (styles)
-      code: `
-      @Component({
-        styles: ['div { display: none; }']
-      })
-      class Test {}
-      `,
-      options: [
-        {
-          styles: -5,
-        },
-      ],
-    },
-    // should succeed if the number of lines does not exceeds the default lines limit (animations)
+    // should succeed if the number of the animations lines does not exceeds the default lines limit
     `
     @Component({
-      animations: [\`state('void', style({opacity: 0, transform: 'scale(1, 0)'}))\`]
+      animations: [state('void', style({opacity: 0, transform: 'scale(1, 0)'}))]
     })
     class Test {}
     `,
-    {
-      // should succeed if a negative limit is used and the number of lines does not exceeds the default lines limit (animations)
-      code: `
-      @Component({
-        animations: [\`state('void', style({opacity: 0, transform: 'scale(1, 0)'}))\`]
-      })
-      class Test {}
-      `,
-      options: [
-        {
-          animations: -5,
-        },
-      ],
-    },
-    // should succeed when none of the template, styles and animations properties are present
+    // should succeed if template, styles and animations properties are not present
     `
     @Component({
       styleUrls: ['./foobar.scss'],
       templateUrl: './foobar.html',
     })
     class Test {}
     `,
+    `
+    @Component({
+      animations: [
+        state('void', style({opacity: 0, transform: 'scale(1, 0)'}))
+      ],
+      templateUrl: './foobar.html',
+    })
+    class Test {}
+    `,
   ],
   invalid: [
     convertAnnotatedSourceToFailureCase({
       description:
-        'should fail if the number of lines exceeds the default lines limit (template)',
+        'should fail if the number of the template lines exceeds the default lines limit',
```

Comment:
Why this change to not highlight the whole template/styles etc? Seems like it would be way easier to miss in an editor this way?

If the concern is visual noise, then surely:

```ts
  template: `...`
  ~~~~~~~~
```

...would be a better way to go?

Reply:
The reported node remains the _property_ value as before. What I did was just a clean up to reduce the visual noise in tests only:

```diff
-        styles: [
-                ~
-          \`
-~~~~~~~~~~~
-            div {
-~~~~~~~~~~~~~~~~~
-              display: block;
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-              height: 40px;
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-            }
-~~~~~~~~~~~~~
-          \`
-~~~~~~~~~~~
-        ]
-~~~~~~~~~

+        styles: [
+               ~
+          \`
+            div {
+              display: block;
+              height: 40px;
+            }
+          \`
+        ]
+        ~
```

Correct me I'm wrong but In tests we don't need to put the character on every line/character to ensure that, we can simply put the character in the begin and other in the end.

Reply:
Ha well if that's the case then that's good, I must have forgotten sorry 😅